src/pages/LoginView.jsx

```diff
@@ -78,13 +95,13 @@ function LoginView() {
               <Input 
                 className='auth'
                 label='Email'
-                onValueChange={(email) => setEmail(email)} 
+                onValueChange={(email) => emailValidationCheck(email)} 
```

Comment:
I don't think the email and password need to be validated for login, since the server is the source of truth. The form can then be validated for empty inputs before submitting, or you can call that form validation function onChange if you want to disable the login button.

Reply:
yeah just check for empty, the server checks the rest

Reply:
Does the backend currently let us know what the error it is to give the best direction? I would think there would be multiple responses needed. One to say 'invalid email' another to say 'email already taken or used'. I also don't understand the purpose of making a request to the backend when it can be handled client side?

Reply:
You usually don't want to give any indication of what the actual error is when logging in. You don't want to give any hints to a malicious user. But I agree that it should be handled client side for registering.

Reply:
Ah I see this is the logged in page and I agree, will remove